lnrpc/lightning.proto

```diff
@@ -1471,7 +1471,7 @@ message Channel {
     height, the next 3 the index within the block, and the last 2 bytes are the
     output index for the channel.
     */
-    uint64 chan_id = 4 [jstype = JS_STRING];
+    uint64 short_chan_id_int = 4 [jstype = JS_STRING];
```

Comment:
we are not renaming stuff, we are deprecating it instead, so no worries @alexbosworth

Reply:
I think we can keep it but instead return the right chan_id not the short chan id here, but I need to think about backwards-compatibility issues.

Reply:
The right chan_id being in string format may also introduce backward compatibility issues.

Reply:
So, I can think of two solutions:  
- Mark `chan_id` as deprecated and introduce fields like `short_chan_id_int`, `short_chan_id`, and `channel_id`.  
- Introduce new names (eg.- `chan_id` for `uint64`, `chan_id_str` for string, and `channel_id` for BOLT-2 definition) for the additional fields while preserving the same name for the existing field.

wdyt?

Reply:
Yeah you cannot reuse this, it is uint64 and for the real channel_id we need string or bytes, so best is to start from a clean state and deprecate the old value.